stable/anchore-engine/README.md

```diff
@@ -609,29 +619,63 @@ anchoreApi:
 
 ## Utilize an Existing Secret
 
-Secrets should be created prior to running `helm install`. These can be used to override the secret provisioned by the Helm chart, preventing plain text passwords in your values.yaml file.
+Rather than passing secrets into the Helm values file directly, users can create secrets in the namespace prior to deploying this Helm chart. When using existing secrets, the chart will load environment variables into deployments from the secret names specified by the following values:
+
+* `.Values.anchoreGlobal.existingSecretName` [default: anchore-engine-env]
+* `.Values.anchoreEnterpriseFeeds.existingSecretName` [default: anchore-enterprise-feeds-env]
+* `.Values.anchoreEnterpriseUi.existingSecretName` [default: anchore-enterprise-ui-env]
+
+To use existing secrets, set the following in your values file:
 
 ```yaml
 anchoreGlobal:
-  # The secret should define the following environment vars:
-  # ANCHORE_ADMIN_PASSWORD
-  # ANCHORE_DB_PASSWORD
-  # ANCHORE_SAML_SECRET (if applicable)
-  existingSecret: "anchore-engine-secrets"
-
-
-anchoreEnterpriseFeeds:
-  # The secret should define the following environment vars:
-  # ANCHORE_ADMIN_PASSWORD
-  # ANCHORE_FEEDS_DB_PASSWORD
-  # ANCHORE_SAML_SECRET (if applicable)
-  existingSecret: "anchore-feeds-secrets"
-
-anchoreEnterpriseUI:
-  # This secret should define the following ENV vars
-  # ANCHORE_APPDB_URI
-  # ANCHORE_REDIS_URI
-  existingSeccret: "anchore-ui-secrets"
+  useExistingSecrets: true
+```
+
+Create the following secrets:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: anchore-engine-env
+type: Opaque
+stringData:
+  ANCHORE_ADMIN_PASSWORD: foobar1234
+  ANCHORE_DB_NAME: anchore
+  ANCHORE_DB_USER: anchoreengine
+  ANCHORE_DB_HOST: anchore-postgresql:5432
```

Comment:
NIT: people might copy/paste this and create the secret but deploy the platform using a different name. I think we should put a note next to the <hosts> to let them know what has to change with the deployment name if not naming the helm deployment 'anchore'

Reply:
Good observation! I'll add a comment to the secret explaining that these secrets will only work as-is if they're using helm deployed postgres/redis with the default values & a release name of `anchore`.